builtin/game/register.lua

```diff
@@ -404,7 +404,11 @@ function core.override_item(name, redefinition)
 end
 
 
-core.callback_origins = {}
+core.callback_origins = setmetatable({}, {
+	__index = function()
+		return {mod = "", name = ""}
```

Comment:
Should I turn this into an upvalue in order to not create a garbage object every time this is called?

Reply:
If we're worrying about the details of a feature that's not even needed may be best to get rid of it for simplicity's sake too :wink: